recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "5.0.3" %}
+{% set version = "5.1" %}
 
 package:
   name: cgal-cpp
   version: {{ version }}
 
 source:
   url: https://github.com/CGAL/cgal/releases/download/releases/CGAL-{{ version }}/CGAL-{{ version }}.tar.xz
-  sha256: e5a3672e35e5e92e3c1b4452cd3c1d554f3177dc512bd98b29edf21866a4288c
+  sha256: 94779e80db978c01768a1b62889ddcfcf74dababa70b73d1e493823d273e0ab7
```

Comment:
Note that the url has changed, because the release tag is `v5.1` and not `releases/CGAL-5.1`:

It should be `https://github.com/CGAL/cgal/releases/download/v{{ version }}/CGAL-{{ version }}.tar.xz` now.

Reply:
Yep, just noticed it as well. On my way to fix it.